docs/en/ingest-management/elastic-agent/scaling-on-kubernetes.asciidoc

```diff
@@ -0,0 +1,220 @@
+- Please follow the steps to run the {agent} on <<running-on-kubernetes-standalone>> page.
+
+
+
+[discrete]
+== Observability under scale
+
+Using https://www.elastic.co/guide/en/welcome-to-elastic/current/getting-started-kubernetes.html[Elastic Observability] to monitor Kubernetes infrastructure at scale, imposes a higher complexity in order to configure and adjust Elastic Stack based on different kubernetes sizes. The Horizontal Pod scaling changes the shape of your Kubernetes workload by automatically increasing or decreasing the number of Pods, thus this will impact the following:
+
+- The amount of metrics being collected from several kubernetes endpoints
+- The agent's resources to cope up with the higher CPU and Memory needed for internal processesing
+- The ElasticSearch resources due to higher rate of metric ingestion needs
+- The Dashboard's Visualisations response times as more data are requested on a given time window 
+
+
+[discrete]
+== Configure Agent Resources
+
+The kubernetes observability is based on https://docs.elastic.co/en/integrations/kubernetes[kubernetes integration], which is fetching metrics from several components:
+
+- **Per node:**
+  - kubelet
+  - controller-manager
+  - scheduler
+  - proxy
+- **Cluster Wide (i.e. correspond to the whole cluster):**
+  - kube-state-metrics
+  - apiserver
+
+The Elastic Agent manifest is deployed by default as daemonset. That said, each elastic agent by default is being deployed on every node of kubernetes cluster. 
+
+Additionally by default, one agent is elected as **leader**(more info <<kubernetes_leaderelection-provider>>) in our https://github.com/elastic/elastic-agent/blob/main/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml#L32[manifests] and this will be responsbile for also collecting the cluster wide metrics
+
+--
+[role="screenshot"]
+image::../images/k8sscaling.png[Agent as daemonset]
+--
+
+Above schema explains how {agent} collect and send metrics to ElasticSearch. Because Leader Agent is responsible to collect also cluster wide metrics this means that requires additional resources.
+
+Daemonset installation simplify the installation of {agent} because we specify less kubernetes resource types inside the manifest and it is the only supported way of <<running-on-kubernetes-managed-by-fleet, managed Elastic Agent scenario>>
+
+Based on our tests, the following table provides guidelines to adjust {agent} limits on diffrent kubernetes sizes:
+
+|===
+| No of Pods in K8s Cluster | Leader Agent Resources | Rest of Agents 
+| 1000   | cpu: "1500m",  memory: "800Mi" | cpu: "300m",  memory: "600Mi" 
+| 3000   | cpu: "2000m",  memory: "1000Mi" | cpu: "400m",  memory: "800Mi" 
+| 5000   | cpu: "3000m",  memory: "2500Mi" | cpu: "500m",  memory: "900Mi" 
+| 10000  | cpu: "3400m",  memory: "4500Mi" | cpu: "700m",  memory: "1000Mi" 
+|===
+
+> Above tests run with {agent} version 8.7 and scraping period of 10sec in Kubernetes Policy
```

Comment:
```suggestion
> The above tests' results came from using {agent} version 8.7 and setting the scraping period to 10sec in the Kubernetes Policy
```

Reply:
Still reviewing this results and will include 8.8 with TSDB  I think only
I